libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -1912,6 +1912,10 @@ namespace SohImGui {
                         "(medallions/stones/songs). Note that these fanfares\n"
                         "are longer than usual."
                     );
+                    PaddedEnhancementCheckbox("Disable Random Rupee Names", "gRandoDisableRandomRupeeNames", true, false);
+                    Tooltip(
+                        "Reverts textboxes for collecting rupees to their default state, disabling the randomization of rupee names."
+                    );
```

Comment:
we should probably make this an enable, not a disable thing. if you want it to be on by default then we can set the cvar default

Reply:
I suppose that's true. Is there an easy way to check if a CVar exists at all instead of checking if it's 0 or non-existing?

Reply:
registercvar i think it's called

Reply:
Perfect. Thank you.